server/src/dtos/duplicate.dto.ts

```diff
@@ -19,7 +19,8 @@ export function mapDuplicateResponse(assets: AssetResponseDto[]): DuplicateRespo
 
   const grouped = groupBy(assets, (a) => a.duplicateId);
 
-  for (const [duplicateId, assets] of Object.entries(grouped)) {
+  for (const [duplicateId, unsortedAssets] of Object.entries(grouped)) {
+    const assets = sortBy(unsortedAssets, (a) => a.localDateTime);
```

Comment:
Very tiny nit pick: can we avoid single letter variables? :)
```suggestion
    const assets = sortBy(unsortedAssets, (asset) => asset.localDateTime);
```